src/pybind/mgr/cephadm/module.py

```diff
@@ -1848,9 +1848,16 @@ def describe_service(self, service_type: Optional[str] = None, service_name: Opt
                 continue
             if service_name is not None and service_name != nm:
                 continue
+
+            if spec.service_type != 'osd':
+                size = spec.placement.get_target_count(self.cache.get_schedulable_hosts())
```

Comment:
```suggestion
                size = spec.placement.get_target_count(self._schedulable_hosts())
```
since https://github.com/ceph/ceph/pull/43331 hasn't been backported

Reply:
and this is why I love static type checking so much:

```
cephadm/__init__.py:1: note: In module imported here:
cephadm/module.py: note: In member "describe_service" of class "CephadmOrchestrator":
cephadm/module.py:1853: error: "HostCache" has no attribute "get_schedulable_hosts"
Found 1 error in 1 file (checked 17 source files)
```